### DISCUSSION TITLE
Unable to add onPress event on SVG Path

### DISCUSSION THREAD

**Discussion by mshakirkhan** [software-mansion/react-native-svg#2194] [Q&A]
Hi, I need help. In React Native CLI I am unable to add an onPress event on Path in my SVG, I am trying to simply Alert when I press the path, I am using the 14.1.0 version, and the React Native version is 0.72.4. Here is my code 

import Svg, { Path, G, Rect, Defs } from "react-native-svg";
import { View, TouchableOpacity, TouchableHighlight,  StyleSheet, Alert } from "react-native";

const SvgComponent = () => {

  const handlePress = () => {
    Alert.alert('Path Pressed!', 'You have pressed the SVG path.');
  }

  return (
    <View style={{ backgroundColor: 'grey', margin: 0, padding: 0, width:'100%', height:400 }}>
      <Svg style={{ margin: 0, padding: 0}} viewBox="0 0 2075 1991" width="100%" height="100%" fill="none">
        <Path
          d="M2072.43 17.367l-4.42 622.727-465.55 5.459 9.91 1261.597-1587.942 7.75L23.08 391.855 14.477 2.784 609.462 7 2072.43 17.367z"
          fill="#EDF5D5"
          stroke="#000"
          strokeWidth={4}
        />
        <TouchableOpacity onPress={handlePress}>
          <Path
              d="M114.143 614.505l126.611 125.9-.617 87.031c.006 1.257-.014 1.967-.014 1.967l.014-1.967c-.037-7.594-1.028-35.182-13.166-47.128l-49.16-47.849-17.117 16.379-47.164-47.835 14.615-15.897-41.198-42.793 27.196-27.808z"
              fill="red"
              stroke="#000"
              data-name="A01"
            />  
        </TouchableOpacity>      
        <Path
          d="M136.369 687.293l-2.058-2.121 22.655-10.071 2.025 2.088-10.754 22.339-2.057-2.121 9.219-18.633-.126-.13-18.904 8.649zm7.284-5.547l8.293 8.548-1.795 1.741-8.292-8.548 1.794-1.741zm11.881 25.76c-1.192-1.229-1.873-2.6-2.043-4.114-.165-1.518.17-3.126 1.005-4.822.84-1.702 2.168-3.434 3.985-5.195 1.805-1.752 3.568-3.019 5.289-3.802 1.731-.782 3.354-1.069 4.867-.858 1.523.211 2.876.925 4.058 2.143 1.181 1.218 1.852 2.589 2.011 4.113.169 1.524-.166 3.137-1.006 4.838-.829 1.702-2.147 3.429-3.952 5.18-1.816 1.762-3.587 3.037-5.314 3.825-1.721.783-3.338 1.069-4.851.859-1.507-.216-2.857-.938-4.049-2.167zm1.794-1.741c1.182 1.218 2.687 1.594 4.515 1.129 1.829-.465 3.831-1.752 6.006-3.862 1.446-1.404 2.528-2.753 3.244-4.05.721-1.291 1.069-2.483 1.042-3.576-.021-1.088-.421-2.034-1.202-2.839-1.171-1.207-2.682-1.573-4.532-1.097-1.845.47-3.852 1.757-6.021 3.862-1.447 1.403-2.528 2.748-3.244 4.033-.716 1.286-1.061 2.47-1.034 3.553.031 1.088.44 2.037 1.226 2.847zm29.476.094L170.1 722.063l-1.963-2.022 14.584-14.148-.095-.098-6.664-1.451 2.055-1.994 6.824 1.486 1.963 2.023z"
          fill="#000"
        />
        <Defs></Defs>
      </Svg>
    </View>
  )
}

const styles = StyleSheet.create({

});

export default SvgComponent

**Comment by bohdanprog** [comment]
Hello @mshakirkhan,
If that issue still persists, could you please open a new issue with a reproduced example, thank you
